src/openrct2/interface/Window.h

```diff
@@ -128,6 +128,8 @@ struct rct_widget
     }
 };
 
+using ViewportCoordsXY = ScreenCoordsXY;
```

Comment:
I think we should try to reduce the amount of coordinate types, we have already a lot of them imho.

Reply:
Would it be clear enough that Contains and ContainsScreen and when to use either?

Reply:
I think documenting the functions should provide a better way to distinguish what needs to be called.

Reply:
Okay so I'll just add a bigger comment stating when to use each of the functions.